base/tools/src/main/java/com/netscape/cmstools/pkcs7/PKCS7CertExportCLI.java

```diff
@@ -51,10 +51,18 @@ public void printHelp() {
     }
 
     public void createOptions() {
-        Option option = new Option(null, "pkcs7-file", true, "PKCS #7 file");
+        Option option = new Option(null, "pkcs7", true, "PKCS #7 file");
```

Comment:
Is this an input file? I'm a little confused (just from reading the help text) what the difference between `pkcs7` file is vs `outfile` -- what format is the output in, if not PKCS#7?

Reply:
The input must be a PKCS7 file (specified with `--pkcs7 <filename>`), but the output can be a series of PEM certs in a single file, or PEM certs in separate files.